app/src/apps/osfi.py

```diff
@@ -45,24 +64,81 @@ def get_pie(data, column):
                 width=12,
             )
         ),
+        dbc.Row(
+            dbc.Col(
+                build_figure_container(
+                    title="Évolution temporelles des émissions (électricité)", id="electricity_time_series", footer="Explications..."
+                ),
+                width=12,
+            )
+        ),
+        dbc.Row(
+            dbc.Col(
+                build_figure_container(
+                    title="Évolution temporelles des émissions (gaz)", id="gaz_time_series", footer="Explications..."
+                ),
+                width=12,
+            )
+        ),
     ]
 )
 
 
 @app.callback(
     [
         Output("osfi-all-data-table", "columns"),
+        Output("osfi-all-data-table", "row_selectable"),
+        Output("osfi-all-data-table", "style_data"),
+        Output("osfi-all-data-table", "style_cell_conditional"),
         Output("osfi-all-data-table", "data"),
-        Output("emission-electricity-pie", "figure"),
-        Output("emission-gas-pie", "figure"),
     ],
     [Input("dashboard-selected-entity", "children")],
 )
+
 def update_graphs(selected_entity):
     service = oc.get_entity_by_id(selected_entity)
     data = oh.get_structure_data(service.code_osfi)
     columns = [{"name": i, "id": i} for i in data.columns]
+    columns = [{"name": i, "id": i} for i in data.loc[:,["Nom du bien"]]]
```

Comment:
Probably the result of some iterations :p
```suggestion
    columns = [{"name": i, "id": i} for i in ["Nom du bien"]]
```
I would actually suggest we keep more information about the buildings (like address) as the `Nom du bien` field seems a bit blurry to understand:
```suggestion
    columns_to_keep = ["Nom du bien", "Building type", "Adresse", "Code postal", "Ville", "Departement"]
    columns = [{"name": i, "id": i} for i in columns_to_keep]
```